composeApp/src/commonMain/kotlin/common/BottomBar.kt

```diff
@@ -0,0 +1,115 @@
+package common
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.runtime.Composable
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import dev.icerock.moko.resources.compose.painterResource
+import instaclone.resources.MR
+
+@Composable
+fun BottomBar() {
```

Comment:
I think you can make this component dynamic, something like:

```kotlin
data class BottomBarItem(
        val isSelected: Boolean, 
        val iconRes: ImageResource,
        val contentDescriptionRes: StringResource,
...
)

@Composable
fun BottomBar(bottomBarItems: List<BottomBarItem>, modifier: Modifier = Modifier) {
     ...
}
```

Reply:
I reorganize the component like so: e6d412d7c92e3237c2683d6a0bff9907fe0b777f
Is this enough?